inventory.go

```diff
@@ -26,37 +51,27 @@ func main() {
 	if err != nil {
 		panic(fmt.Errorf("Fatal error config file: %s \n", err))
 	}
-	props := props{rackhdUrl: config.GetString("rackhd_api_url"), groups: config.GetStringSlice("groups")}
-
-	args := CLIArgs{}
-	arg.MustParse(&args)
-	if args.List {
-		handleList(props)
-		return
-	}
-
-	handleList(props)
-
-	return
+	return props{rackhdUrl: config.GetString("rackhd_api_url"), groups: config.GetStringSlice("groups"), filterGroup: config.GetString("filter_group")}
```

Comment:
better to split this into multiple lines. Easier to read vertically than horizontally, two rules:

- don't have lines with more than 120 cols
- if it has more than 2 arguments, split into multiple lines